src/LEGO.AsyncAPI/Models/AsyncApiComponents.cs

```diff
@@ -66,12 +66,12 @@ public class AsyncApiComponents : IAsyncApiExtensible, IAsyncApiSerializable
         /// <summary>
         /// An object to hold reusable Operation Reply Objects.
         /// </summary>
-        public IDictionary<string, AsyncApiReply> Replies { get; set; } = new Dictionary<string, AsyncApiReply>();
+        public IDictionary<string, AsyncApiOperationReply> Replies { get; set; } = new Dictionary<string, AsyncApiOperationReply>();
```

Comment:
you are missing the schemas change. Its now MultiFormat | SchemaObject.
So in this lib it would be multiformat always.

Reply:
Also missing ExternalDocs